modules/auxiliary/gather/browser_sop_bypass.rb

```diff
@@ -0,0 +1,53 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Browser SOP Bypass",
+        'Description'    => %q(
+          This module do a server-redirect combined with a data-URI end up bypassing the Same Origin Policy,which leads to all kind of vulnerabilities like stealing user passwords.
```

Comment:
Please wrap your lines on the description to be sensible. But regardless, the given description doesn't really describe what the module actually does. Right now, it's super unclear what the user should expect by running this module.